client/layout/guided-tours/config-elements.js

```diff
@@ -63,20 +70,36 @@ export class Tour extends Component {
 
 	setTourMeta( props ) {
 		const { branching, next, quit, isValid, name, version, stepName } = props;
-		this.tourMeta = { branching, next, quit, isValid, tour: name, tourVersion: version, step: stepName };
+		this.tourMeta = {
+			branching,
+			next,
+			quit,
+			isValid,
+			tour: name,
+			tourVersion: version,
+			step: stepName,
+			isLastStep: this.isLastStep( { step: stepName, branching } ),
+		};
 	}
 
-	render() {
+	findNextStep() {
 		const { children, stepName } = this.props;
-		const nextStep = find( children, stepComponent =>
+		return find( children, stepComponent =>
```

Comment:
By the way, ES6 syntax will always require braces if there are more than one argument in such a form, or if the argument is more complex. Examples:

- `( a, b ) => 'foo'` 
- `( { a } ) => 'foo'`
- `( [ a ] )` => 'foo'`